File: DetailViewTabCount/Module/VB/DetailViewTabCountVB.Module/Helpers/DetailViewControllerHelper.vb
Public Module DetailViewControllerHelper
	Public Function ClearItemCountInTabCaption(ByVal caption As String) As String
		Dim index As Integer = caption.IndexOf("("c)
		If index <> -1 Then
			Return caption.Remove(index - 1)
		End If
		Return caption
	End Function
	Public Function AddItemCountToTabCaption(ByVal caption As String, ByVal count As Integer) As String
		Return $"{caption} ({count})"
	End Function
End Module

File: CS/XPO/TabCount/TabCount.Module/DetailViewControllerHelper.cs
namespace TabCount.Module
{
    public static class DetailViewControllerHelper
    {
        public static string ClearItemCountInTabCaption(string caption)
        {
            int index = caption.IndexOf('(');
            if (index != -1)
            {
                return caption.Remove(index - 1);
            }
            return caption;
        }

        public static string AddItemCountToTabCaption(string caption, int count)
        {
            return $"{caption} ({count})";
        }
    }
}
